Automattic/wp-calypso
# Marketplace: Add new Plugin categories to improve SEO

Resolves https://github.com/Automattic/wp-calypso/issues/76009

## Proposed Changes

* Add categories suggested in pdh6GB-2M1-p2#comment-3577 

## Testing Instructions



* Navigate to `/plugins` page 
* In the categories, expand the `More` category
* Check that the categories in the spreadsheet found here pdh6GB-2M1-p2#comment-3577 are listed at the bottom of the list. Please note that `Donations` existed as `Crowdfunding` and `Analytics` was already there.
* Select each and every new category and make sure:
  * Results are displayed
  * The category title displayed under the categories menu and on top of the Plugin results is correct 

|Before | After Page 1| After Page 2| After Page 3|
|-------|------|------|------|
| ![CleanShot 2023-05-02 at 15 53 56@2x](https://user-images.githubusercontent.com/3519124/235688246-33007873-b89e-4f0b-af50-d2f95311cb6e.png) | ![CleanShot 2023-05-02 at 15 54 42@2x](https://user-images.githubusercontent.com/3519124/235688331-a7692c6e-7d27-44f2-9911-2c9c9e72c147.png) | ![CleanShot 2023-05-02 at 15 54 50@2x](https://user-images.githubusercontent.com/3519124/235688367-d9e0be88-55a1-449d-9710-f720b1844504.png) | ![CleanShot 2023-05-02 at 15 54 55@2x](https://user-images.githubusercontent.com/3519124/235688403-49d05700-7b74-444f-a154-4e5f81b26ba3.png) |

## Pre-merge Checklist
